src/app/pages/OpenWalletPage/index.tsx

```diff
@@ -79,9 +79,7 @@ export function SelectOpenMethod() {
           i18nKey="openWallet.bitpie.warning"
           t={t}
           defaults="❗ BitPie wallet users: You cannot import the mnemonic phrase directly from your BitPie wallet. <0>Check documentation for details.</0>"
-          components={[
```

Comment:
Should we keep the link to a particular question in case FAQ grows in the future?

Reply:
(I don't care too much, but if we do: Can anchor be stable if heading changes in the future?)

Reply:
Unfortunately, having a stable anchor is not possible with GitBook.